contrib/nydusify/pkg/commiter/nsenter.go

```diff
@@ -0,0 +1,183 @@
+package commiter
```

Comment:
Should put the copyright, the codes ported from `https://github.com/Devatoria/go-nsenter`.